packages/reports/addon/templates/components/navi-column-config/metric.hbs

```diff
@@ -7,7 +7,8 @@
       @triggerId="{{this.classId}}-columnParameter" 
       @dropdownClass="navi-column-config-metric__parameter-dropdown"
       @selected={{this.currentParameter}}
-      @searchPlaceholder="Type to search"
+      @searchField="id"
+      @searchPlaceholder="Type to search by id"
```

Comment:
Can this be `Search ${paramName}`?

Reply:
What does paramName refer to, `paramName = Revenue` or `paramName = description`?